sec_certs/model/dependency_finder.py

```diff
@@ -0,0 +1,117 @@
+from typing import List, Set, Dict, Tuple
+
+
+class DependencyFinder:
+
+    def __init__(self):
+        self.dependencies = {}
+
+    @staticmethod
+    def _update_direct_references(referenced_by: Dict, cert_id: str, this_cert_id: str) -> None:
```

Comment:
General note to typehints (applicable on other places in the commits). Please, make the typehints as aspecific as possible. E.g. I assume that `referenced_by` has to be `Dict[str]` for the function to properly work (or any other type instead of `str`). If that is the case, please write `Dict[str]` instead of `Dict` to provide more guidance.